tests/core/snapd-maintenance-msg/task.yaml

```diff
@@ -0,0 +1,52 @@
+summary: Verify that the maintenance message is included in the daemon API responses when available
+
+# kinda slow test, probably enough to keep on one system
+systems: [ubuntu-core-20-64]
+
+prepare: |
+    snap install http jq
+
+restore: |
+    snap remove http jq
```

Comment:
we try not to use http anymore in our own tests but curl --unix

Reply:
switched to curl